src/x/xRolling.js

```diff
@@ -0,0 +1,26 @@
+import { xPadding } from './xPadding';
+/**
+ * This function calculates a rolling average
```

Comment:
Maybe one should write in the docs a note that in the default setting the returned array will be smaller.